### ISSUE TITLE
Debugging with multiple pilets only uses the first found pilet's peer dependencies

### ISSUE THREAD

**Issue by nolanlocke** [smapiot/piral#241]
# Bug Report

## Prerequisites

- [ ] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version
piral-cli@next

## Description
When running `pilet debug` on multiple pilets, only peer dependencies listed in the first found pilet will be exposed to the other pilets.

## Expected behavior
Externals are gathered from all pilets rather than the first found

## Actual behavior
Only the peer dependencies of the first pilet found in the glob are used for externals.

## Possible Origin / Solution
I believe the solution would be concating all externals (and removing duplicate values) from the pilet passed in [here](https://github.com/smapiot/piral/blob/b819942e913b703141154b98ad648faa4ff2abbf/src/tooling/piral-cli/src/apps/debug-pilet.ts#L154)

**Comment by nolanlocke** [comment]
I may have had a fundamental misunderstanding - If `pilet upgrade` will go through and update the peerDependencies it's likely I just need to properly use that command

**Comment by FlorianRappl** [comment]
Yeah that is what `pilet upgrade` should do. It should essentially make sure your `package.json` (and the rest) of the pilet changes to whatever the Piral instance "wants" / "offers" / "expects".

Its a helper that makes sure you will produce a pilet that is fully compatible (and can leverage all the possibilities) with your shell.
